src/yaml.rs

```diff
@@ -44,6 +53,8 @@ pub enum Yaml {
     Hash(self::Hash),
     /// Alias, not fully supported yet.
     Alias(usize),
+    /// Yaml node with block or line comments
+    CommentedYaml(self::CommentedYaml),
```

Comment:
this is a breaking change, so I guess, it'll require bumping minor version (0.4.5 to 0.5.0), maybe? But I'm not the maintainer :)

Reply:
Since this is a breaking change, it might also potentially be worth adding [non_exhaustic](https://doc.rust-lang.org/reference/attributes/type_system.html) to the enum, so in the future changes like this _won't_ be breaking.